paywall/src/components/content/CheckoutContent.tsx

```diff
@@ -230,7 +230,7 @@ export default function CheckoutContent() {
     return (
       <Greyout>
         <MessageBox>
-          <p>Please check your browser wallet to complete the transaction.</p>
+          <p>Please check your browser wallet.</p>
```

Comment:
This particular use of the message only ever happens during a transaction, though I would be in favor of pulling this out and using the other component (which I forgot we moved into the paywall) with a parameter as mentioned in my other comment.